tests/unit/test-strings.cpp

```diff
@@ -116,6 +115,11 @@ main (int __attr_unused___ argc,
     lit_utf8_size_t cesu8_string_size = (i == 0) ? 0 : (lit_utf8_size_t) (rand () % max_bytes_in_string);
     ecma_length_t length = generate_cesu8_string (cesu8_string, cesu8_string_size);
 
+    ecma_string_t *char_collection_string_p = ecma_new_ecma_string_from_utf8 (cesu8_string, cesu8_string_size);
```

Comment:
@huxinx, as now `ecma` component is involved, the function should initialize it with `ecma_init`. Also, seems that `lit_init` is missing.

Please, add `lit_init` and `ecma_init` to the beginning of the function like in `jerry_init`, and corresponding `ecma_finalize`, `lit_finalize`, like in `jerry_cleanup`.